README.md

```diff
@@ -1,6 +1,27 @@
 > [!IMPORTANT]  
 > The code in this repository and its dependencies are still under audit. It is not yet recommended for production use.
 
+# Getting Started
```

Comment:
Thanks! I think I kinda prefer "Developing" 🤔 as it might be confused for getting started with a smart account?

Reply:
That makes sense. I'll update the PR. Is there anything else that you think might be worthwhile to include in a section like that?

From an external developer perspective I think some more insights/references into the unique PassKey approach enabled through the wallet be super helpful. It's a pretty wild to see my account created on www.smart-wallet.xyz and suddenly be available in wallet-beta.coinbase.com too. I immediately want to know more about that.